src/main/scala/caravan/bus/tilelink/Harness.scala

```diff
@@ -0,0 +1,79 @@
+package caravan.bus.tileink
```

Comment:
the package name should be `caravan.bus.tilelink` . There is a typo in the spelling of tilelink